cmd/snap-repair/cmd_run_test.go

```diff
@@ -33,8 +33,22 @@ import (
 	"github.com/snapcore/snapd/release"
 )
 
+func (r *repairSuite) TestNonRoot(c *C) {
+	restore := repair.MockOsGetuid(func() int { return 1000 })
+	defer restore()
+	restore = release.MockOnClassic(false)
+	defer restore()
+
+	os.Args = []string{"snap-repair", "run"}
+	err := repair.Run()
```

Comment:
Should there be a `defer func() { os.Args = origArgs }()` right above this line?

Reply:
thank you! nice catch.